.github/workflows/linters.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Lighthouse
         run: npm install -g @lhci/cli@0.7.x
       - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=./dist.
```

Comment:
I think it'll be good to add use the original linter files, to ensure that we are following the project requirements.